homeassistant/components/homeconnect/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "homeconnect",
```

Comment:
Our standard is to separate words with underscore. If the name of the brand is Home Connect we should name the integration `"home_connect"`.

Reply:
Oh ... OK.